src/atoms.ts

```diff
@@ -36,8 +36,8 @@ function parseAtom(
 
             const { html, css, mainJS: js } = atom?.data?.interactive;
 
-            if (!html || !css) {
-                return err(`No html or css for atom: ${id}`);
+            if (!html && !css && !js) {
```

Comment:
So *all three* fields have to be missing to prevent a render? If for example it _just_ had CSS it would still try to render?

Reply:
I think it's unlikely people would use just css but maybe not impossible https://a.singlediv.com/